cockatrice/src/tappedout_interface.cpp

```diff
@@ -28,9 +28,17 @@ void TappedOutInterface::queryFinished(QNetworkReply *reply)
 
     if(reply->hasRawHeader("Location"))
     {
+        /*
+         * If the reply contains a "Location" header, a relative URL to the deck on TappedOut
+         * can be extracted from the header. The http status is a 302 "redirect".
+         */
         QString deckUrl = reply->rawHeader("Location");
         QDesktopServices::openUrl("http://tappedout.net" + deckUrl);        
     } else {
+        /*
+         * Otherwise, the deck has not been parsed correctly. Error messages can be extracted
```

Comment:
> Is extracting the messages a future improvement item?

It' already implemented a few lines of code later. The error messages are extracted from the html and shown to the user in a messagebox